shell-color-themes/windows/windowsterminal/dark.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Hey! Since the folder is already named `windows` I think we should rename `windowsterminal` to `terminal` ?

Wdyt 🤔

Reply:
I think that's a good catch, thank you!